icu4c/source/test/testdata/BUILDRULES.py

```diff
@@ -10,6 +10,7 @@ def generate(config, io, common_vars):
     requests += generate_rb(config, io, common_vars)
     requests += generate_sprep(config, io, common_vars)
     requests += generate_conv(config, io, common_vars)
+    requests += generate_lstm(config, io, common_vars)
```

Comment:
Please merge main into your branch so that these diffs go away

Reply:
Thie is the BUILDRULES.py under test/testdata not the one under data/